pom.xml

```diff
@@ -121,7 +121,7 @@
     <collections.version>3.2.2</collections.version>
     <jodatime.version>2.10.5</jodatime.version>
     <joni.version>2.1.31</joni.version>
-    <omid.version>1.0.2</omid.version>
+    <omid.version>1.1.0-SNAPSHOT</omid.version>
```

Comment:
I think this version should be updated to 1.1.0 after we have the omid released.

Reply:
That's correct, Richard.